### DISCUSSION TITLE
How to access hyperparameters for individual models for a TimeSeriesPredictor

### DISCUSSION THREAD

**Discussion by shashankumar2812** [autogluon/autogluon#4072] [Q&A]
Hi,

I ran a hyperparametertuning job using autogluon TimeSeriesPredictor. How do I extract hyperparameters for individual models which were built by the hyperparametertuning job?

**Comment by shchur** [comment]
Hi @shashankumar2812, thank you for the question and sorry for the slow response. Here is a workaround for obtaining this information: https://github.com/autogluon/autogluon/issues/4152#issuecomment-2085256152. 

We will expose this functionality through the API in the v1.2 release (issue for tracking progress: #4152).
